src/GovUk.Education.ExploreEducationStatistics.Data.Model/Observation.cs

```diff
@@ -28,6 +29,7 @@ public class Observation
 
         public ICollection<ObservationFilterItem> FilterItems { get; set; }
 
+        [NotMapped]
```

Comment:
I didn't give this PR a proper review yet as it's still a draft (and I'm not working on the project! 🙂) but just wondering if this is a mistake that you want to remove or if its intentional because we don't need to pull this value back every time we retrieve an Observation?

Reply:
I added this in relation to the bug of editing a MethodologyNote - when you try to update a MethodologyNote in Admin, you get this error:
```
fail: Microsoft.EntityFrameworkCore.Query[10100]
      An exception occurred while iterating over the results of a query for context type 'GovUk.Education.ExploreEducationStatistics.Data.Model.Database.StatisticsDbContext'.
      System.InvalidOperationException: The required column 'CsvRow' was not present in the results of a 'FromSql' operation.
```

Adding `NotMapped` removed this. But I think you're right to highlight this because I don't think this is the actual solution - the solution is for the stored procedure to return a CsvRow value. I'll remove the `NotMapped`

Reply:
Now I've removed it, it's not failing anymore - anyway this is related to the stored procedure that Duncan is going to remove anyway, so it's no an issue.

Reply:
Ok turns out it is an issue - I've updated the stored procedure change in the description with the fix.